src/aks-preview/HISTORY.rst

```diff
@@ -12,6 +12,11 @@ To release a new version, please select a new version number (usually plus 1 to
 Pending
 +++++++
 
+2.0.0b5
```

Comment:
Please update `VERSION` in `setup.py` so the change could be released.

Reply:
Done